hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -86,15 +127,73 @@ The following covergroups have been developed to prove that the test intent has
 #### Scoreboard
 The `sram_ctrl_scoreboard` is primarily used for end to end checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-* analysis port1:
-* analysis port2:
-<!-- explain inputs monitored, flow of data and outputs checked -->
+* tl_a_chan_fifo: TL address channel for CSR accesses
+* tl_d_chan_fifo: TL data channel for CSR accesses
+* alert_fifos: Alert handshakes
+* sram_tl_a_chan_fifo: TL address channel for memory accesses
+* sram_tl_d_chan_fifo: TL data channel for memory accesses
+* kdi_fifo: For key refresh operations from OTP
```

Comment:
This is super nit: but I think we call it OTP_CTRL instead of OTP. (OTP can refer to the actual OTP memory)

Reply:
good point let me update this, thanks!